### PR TITLE
Add symfony config docs

### PR DESCRIPTION
**Pull request by Jean85**
~WIP: this includes #793, do not merge before it.~ #793 closed in favor of this.

This PR expands #793, adding information about Symfony's integration configuration.

### CODE REVIEW

**Comment by Jean85** [comment]
@HazAT that is already linked multiple times in the page. I've added an other link in the customization paragraph nonetheless.

**Comment by Jean85** [comment]
I've rebased on top of the current master, it should fix the build I hope.

**Comment by Jean85** [comment]
Can we merge this one?

**Comment by EvanPurkhiser** [comment]
Why haven't we merged this @HazAT ?

**Comment by Jean85** [comment]
I've updated since in the meantime I've released the stable version.

**Comment by Jean85** [comment]
Can we merge this? Users are getting mistaken with the outdated docs: getsentry/sentry-symfony#221

File: src/collections/_documentation/platforms/php/symfony.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+Below you can find additional documentation that is specific to the bundle usage, or information about the sensible default
+values that are used in some cases.
+
+#### before_breadcrumb and before_send
+The `before_breadcrumb` and `before_send` options both accept a `callable`; thus, you cannot provide it directly through
+ a YAML file; the bundle accepts a service reference (starting with `@`), which you can build in your DIC container.
+
+#### environment
+The `environment` option defaults to the same environment of your Symfony application (i.e. `dev`, `test`, `prod`...).
```
Comment (MimiDumpling) [inline src/collections/_documentation/platforms/php/symfony.md]:
```suggestion
The `environment` option defaults to the same environment of your Symfony application (For example,`dev`, `test`, `prod`...).
```
```suggestion
The environment option defaults to the same environment of your Symfony application (For example, dev, test, prod, etc...).
```

File: src/collections/_documentation/platforms/php/symfony.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+The `in_app_exclude` option is used to mark files as non belonging to your app's source code in events' stack traces.
+In this bundle it has two default values: 
+ * `%kernel.cache_dir%`, to exclude Symfony's cache dir
+ * `%kernel.project_dir%/vendor`, to exclude Composer's dependencies
+
+#### project_root
+The `project_root` options defaults to `%kernel.project_dir%`.
+
+### Listener priorities
+The `listener_priorities` options are used to change the priority at which the bundle's listener are called. They all default
```
Comment (MimiDumpling) [inline src/collections/_documentation/platforms/php/symfony.md]:
```suggestion
The `listener_priorities` options are used to change the priority at which the bundle's listener is called. They all default
```

**Comment by Jean85** [comment]
Thanks, I've applied all the suggestions!

**Pull request merged**